datafusion/src/catalog/schema.rs

```diff
@@ -101,4 +114,70 @@ impl SchemaProvider for MemorySchemaProvider {
         let mut tables = self.tables.write().unwrap();
         Ok(tables.remove(name))
     }
+
+    fn table_exist(&self, name: &str) -> bool {
```

Comment:
nit: we can keep the same as above, use `#[allow(unused_variables)]` instead of `_`.